modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -31,6 +31,7 @@ HashTSDFVolume::HashTSDFVolume(float _voxelSize, cv::Matx44f _pose, float _rayca
     maxWeight(_maxWeight),
     truncateThreshold(_truncateThreshold),
     volumeUnitResolution(_volumeUnitRes),
+    volumeUnitResolutions(_volumeUnitRes, _volumeUnitRes, _volumeUnitRes),
```

Comment:
The same: please update it with code changes

Reply:
done